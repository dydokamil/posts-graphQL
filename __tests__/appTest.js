const request = require('supertest')
const app = require('../app')

describe('app', () => {
  it('should return the status', async () => {
    const response = await request(app)
      .post('/graphql')
      .send({ query: 'query { status }' })
    expect(response.statusCode).toBe(200)
    expect(response.body.data.status).toMatchSnapshot()
  })
})
