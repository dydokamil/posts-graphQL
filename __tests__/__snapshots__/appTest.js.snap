// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app should return the status 1`] = `"GraphQL status: OK"`;
